web/Cargo.toml

```diff
@@ -15,6 +15,7 @@ chrono = { version = "0.4", features = ["wasmbind"] }
 gloo-timers = "0.2"
 http = "0.2"
 js-sys = "0.3"
+names = "0.12"
```

Comment:
Cool - so it works now out-of-the-box. Gotta replace it in fledger, too ;)

Reply:
Indeed, very easy to use, thanks for the link :)